src/main/java/org/eclipse/xpanse/terraform/boot/logging/CustomRequestIdGenerator.java

```diff
@@ -16,10 +16,12 @@
  */
 public class CustomRequestIdGenerator implements CorrelationId {
 
+    public static final String REQUEST_ID = "REQUEST_ID";
+
     @Override
     public String generate(@NonNull HttpRequest request) {
         String uuid = UUID.randomUUID().toString();
-        MDC.put("TASK_ID", uuid);
+        MDC.put(REQUEST_ID, uuid);
```

Comment:
we are replacing this again with the REQUEST_ID in the controller?
This will make the logging not consistent. We must always log all entries of the request with same REQUEST_ID

Reply:
No. Only use the same constant name.